modules/core/src/model/ConsentLanguages.ts

```diff
@@ -38,12 +39,15 @@ export class ConsentLanguages {
     'RU',
     'SK',
     'SL',
+    'SQ',
     'SR-LATN',
     'SR-CYRL',
     'SV',
+    'SW',
+    'TH',
     'TL',
     'TR',
-    'UK',
```

Comment:
@HeinzBaumann Any reason for this removal?

Reply:
Oops. I fix it shortly.